src/Microsoft.AspNet.Mvc.Core/ModelBinding/JQueryFormValueProviderFactory.cs

```diff
@@ -21,10 +21,12 @@ public class JQueryFormValueProviderFactory : IValueProviderFactory
 
             if (request.HasFormContentType)
             {
+                var culture = CultureInfo.CurrentCulture;
```

Comment:
Want to use thread culture here? Request culture, as per https://github.com/aspnet/Mvc/pull/3059/files#diff-fbc3dd5ae10027009ad0c93d673e1141R19 may be more relevant?